example/android/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         minSdkVersion = 21
         compileSdkVersion = 30
         targetSdkVersion = 30
-        ndkVersion = "22.0.7026061"
+        ndkVersion = "20.1.5948944"
```

Comment:
Tried to use `22.0.7026061` (same version as VisionCamera) but this somehow caused builds to fail at the `:react-native-vision-camera:externalNativeBuildDebug` task with `fatal error: 'boost/operators.hpp' file not found`.

Reply:
The official react native template uses `21.4.7075529`, maybe we should downgrade to that?

https://github.com/facebook/react-native/blob/f51773ecdedbac19d25eb20894e532edef2cb304/template/android/build.gradle#L9

Reply:
That's for 0.65 though isn't it? As far as I can tell [0.64.x uses the version I used here](https://github.com/facebook/react-native/blob/0.64-stable/template/android/build.gradle).

Reply:
Oh yeah makes sense. 👍

Reply:
So VisionCamera should probably use the version the actual app's project uses (`project.ext.ndkVersion` or smth like that)